apps/web-giddh/src/app/invoice/generate/invoice.generate.component.html

```diff
@@ -267,7 +267,7 @@
                 </td>
 
                 <td class="text-right" data-title="Total">{{item.total.amount | giddhCurrency}}</td>
-                <td data-title="Discription">{{ item.description ? item.description : '-' }}</td>
+                <td data-title="Discription" class="pending-table-discription">{{ item.description ? item.description : '-' }}</td>
```

Comment:
Please correct spelling of discription in data-title and css class

Reply:
updated